grep/grep.rb

```diff
@@ -1,169 +1,138 @@
-class Grep
+# frozen_string_literal: true
 
-attr_reader :word, :flags, :files
+# Class for search words in files.
+class Grep
+  attr_reader :word, :flags, :files
 
-def initialize(word, flags, files)
-	@word = word
-	@flags = flags
-	@files = files
-end
+  def initialize(word, flags, files)
+    @word = word
+    @flags = flags
+    @files = files
+  end
 
+  def grep
+    found = []
+    files.each do |file|
+      File.readlines(file).each_with_index do |line, index|
+        how_many_flags?(index, line, found, file)
```

Comment:
Method ended with ? should return boolean value. Methods should be named as action.